language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -272,8 +283,10 @@ class CompletionTest {
           |  import java.util.{HashMap => HashMap2}
           |  val x: Hash$m1
           |}""".withSource
-      .completion(m1, Set(("HashMap", Class, "class and object HashMap"),
-                          ("HashMap2", Class, "class and object HashMap")))
+      .completion(m1, Set(("HashMap", Class, "java.util.HashMap"),
+                          ("HashMap", Module, "java.util.HashMap$"),
+                          ("HashMap2", Class, "java.util.HashMap"),
+                          ("HashMap2", Module, "java.util.HashMap$")))
```

Comment:
Might be an orthogonal issue: for a module, it is more friendly to show `java.util.HashMap` instead of `java.util.HashMap$`.

It can be addressed in a different PR.

Reply:
Yeah, but that's because of how `.show` works by default. An even more enigmatic example is 
```
code"class Foo { val foo: Runn${m1} }".withSource
      .completion(m1, Set(
        ("Runnable", Class, "java.lang.Runnable"),
        ("Runnable", Module, "Runnable$")
      ))
```
but that's definitely a separate issue